api/contrib/envoy/extensions/network/connection_balance/dlb/v3alpha/dlb.proto

```diff
@@ -30,4 +30,8 @@ message Dlb {
   // The ID of the Dlb hardware, start from 0.
   // If not specified, use the first available device as default.
   uint32 id = 1;
+
+  // Control how many times Envoy makes an attempt to send event to DLB hardware.
+  // No retry as default.
+  uint32 most_send_retries = 2;
```

Comment:
Sorry, re-read this doc and I think it doesn't reflect the semantics of the field, because the field counts retries, and the comment says attempts.
The number of attempts equals the number of retries + 1.

The field name should be either `max_attempts` or `max_retries`, and the comment and implementation should reflect its semantics.

Reply:
Updated.